src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 16px;
+
+  &__product {
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-top: 16px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin-top: 40px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-1;
```

Comment:
CL11 - use variables for the main values so that you'll be able to reuse them and give them descriptive names. It is better to specify a specific color in a variable, for example `$color-accent-dark-blue` or `$main-text-color`